### PR TITLE
resolves #39 document attributes and options

### LINKED ISSUE #39: Document configuration options
Bits and pieces I learned in discussions that should be documented:

**General diagram options**
> here: https://github.com/asciidoctor/asciidoctor-diagram
> we are trying to stay compatible with Asciidoctor diagram

> please note that diagram attributes are not supported since the Kroki API does not support specific attributes: https://github.com/asciidoctor/asciidoctor-diagram#diagram-attributes

**All Asciidoctor Kroki parameters**

Besides `kroki-server-url` and `kroki-fetch-diagram` there are probably others?

### PR DESCRIPTION
**Pull request by Mogztter**
As always, review is highly appreciated :smile: 

Ping @eshepelyuk @danyill @marcelstoer 

resolves #39

### CODE REVIEW

**Comment by Mogztter** [comment]
Related issue #72
I mention both the block and macro forms. Maybe we should add a dedicated section before to explain the two forms? What do you think?

File: README.md [hunk 1]
```diff
@@ -175,6 +175,109 @@ include::alice-bob.puml[]
...
+....
+graph TD;
+    A-->B;
+    A-->C;
+    B-->D;
+    C-->D;
+....
+```
+
+In the above example,
```
Comment (marcelstoer) [inline README.md:203]: This renders as a single line despite the line breaks here (standard Markdown behavior). If that's unintended you need to add line breaks or define a list.

**Comment by marcelstoer** [comment]
That's fantastic! Thank you.

> Maybe we should add a dedicated section before to explain the two forms? What do you think?

Either that or link to explanations elsewhere.

File: README.md (hunk 1 above)
Comment (Mogztter) [reply README.md:203]: Yes this is intended, I find it easier to edit.

**Pull request merged**